# HG changeset patch
# User andrew
# Date 1369226882 -3600
#      Wed May 22 13:48:02 2013 +0100
# Node ID f559fadbf49130bcb703fa7af38070a74887a6f5
# Parent  b61632814be29cb9cf7fb5fe817a6e45c5729835
8015087: Provide debugging information for programs
Summary: Add missing debug info to unpack200 and jexec
Reviewed-by: erikj

diff --git a/makefiles/CompileLaunchers.gmk b/makefiles/CompileLaunchers.gmk
--- openjdk/jdk/makefiles/CompileLaunchers.gmk
+++ openjdk/jdk/makefiles/CompileLaunchers.gmk
@@ -472,6 +472,7 @@
 			  -D "JDK_FNAME=unpack200.exe" \
 			  -D "JDK_INTERNAL_NAME=unpack200" \
 			  -D "JDK_FTYPE=0x1L",\
+	        DEBUG_SYMBOLS:=true,\
 		MANIFEST:=$(JDK_TOPDIR)/src/windows/resource/unpack200_proto.exe.manifest))
 
 ifeq ($(OPENJDK_TARGET_OS),windows)
@@ -555,6 +556,7 @@
 			 $(call SET_SHARED_LIBRARY_NAME,$(LIBRARY_PREFIX)$(SHARED_LIBRARY_SUFFIX)), \
 		OBJECT_DIR:=$(JDK_OUTPUTDIR)/objs/jexec_obj,\
 		OUTPUT_DIR:=$(BUILD_JEXEC_DST_DIR),\
+		DEBUG_SYMBOLS:=true,\
 		PROGRAM:=jexec))
 
 	BUILD_LAUNCHERS += $(BUILD_JEXEC)
